agent/command/results_native.go

```diff
@@ -70,9 +70,6 @@ type attachResults struct {
 func attachResultsFactory() Command   { return &attachResults{} }
 func (c *attachResults) Name() string { return evergreen.AttachResultsCommandName }
 
-// ParseParams decodes the S3 push command parameters that are
```

Comment:
could we keep this comment and just reword for attach results?

Reply:
I was cleaning up the comments since only these commands document this function. I'm okay with documenting it but do you want me to add it to all the other commands?

Reply:
nah either works, i feel like comments are always helpful so removing them doesn't seem necessary, but you don't need to go do a ton of work here; if you'd rather just keep it removed that's fine too